test/chartFactory.js

```diff
@@ -16,6 +16,17 @@
                     );
                 });
             },
+            "long-label": function (callback) {
+                require(["test/DataFactory", "src/chart/Column"], function (DataFactory, Column) {
+                    callback(new Column()
+                        .columns(DataFactory.ND.subjects.columns)
+                        .data(DataFactory.ND.subjects["data-long-label"])
```

Comment:
FYI - You can inline the data here - you only want it in the factory file if its shared between two samples.